Header header
int32 VescId
int32 ERPM
float32 Current
float32 DutyCycle
float32 AhUsed
float32 AhCharged
float32 WhUsed
float32 WhCharged
float32 TempFet
float32 TempMotor
float32 CurrentIn
float32 PidPos
float32 Tachometer
float32 VoltsIn
float32 ADC1
float32 ADC2
float32 ADC3
float32 PPM